ci-operator/config/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-main__periodic-4.13-upgrade-from-stable-4.12.yaml

```diff
@@ -27,7 +27,7 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: e2e-openstack-upgrade
+- as: e2e-openstack-sdn-upgrade
```

Comment:
Why is sdn being added to the title?
The jobs are configured with ovn-k by default in 4.12. Here is the list of job runs https://prow.ci.openshift.org/job-history/gs/origin-ci-test/logs/periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.13-upgrade-from-stable-4.12-e2e-openstack-upgrade

Reply:
good catch. thank you. I've updated it to use -ovn- instead of -sdn-